### DISCUSSION TITLE
Need a sponsor of the packet imsprog

### DISCUSSION THREAD

**Discussion by bigbigmdm** [linuxmint/discussions#308] [Ideas]
This packet has no analogs. 
Debian [#1057386](https://bugs.debian.org/1057386)
[imsprog-mentors](https://mentors.debian.net/package/imsprog/)
[imsprog-launchpad](https://launchpad.net/~bigmdm/+archive/ubuntu/imsprog)
[imsprog-GitHub](https://github.com/bigbigmdm/IMSProg)

**Comment by Fantu** [comment]
Need to find a Debian developer that sponsor it for try inclusion in Debian (I'm only a Debian Maintainer and I can't upload new packages), if it will be included before [Ubuntu 24.04 feature freeze](https://launchpad.net/ubuntu/+milestone/ubuntu-24.04-feature-freeze) also next Ubuntu will take it with autosync and next Mint based on it.
There aren't DD in Mint team FWIK, so probably this discussion don't help, the only possibility would be only if there are users who are or users who ask for it.

**Comment by bigbigmdm** [comment]
@Fabio, can you tell me how to do this?
All my attempts to post to debian-devel and ubuntu-devel irc are now unresponsive....

**Comment by Fantu** [comment]
Maybe your difficulties with even some basic things after explanations have discouraged any DDs who don't have much time.
Even if the packaging of this software is not big it requires time to check and some responsibility is required: https://wiki.debian.org/SponsorChecklist
You've made some improvements, I don't know if they'll be enough, try to new upload and RFS as described [here](https://github.com/bigbigmdm/IMSProg/issues/19#issuecomment-1914447047) and if this time you'll do all correct maybe you have some chance.
And after you can try to ask again to debian-mentors in IRC.

o.t.: the mention to me is wrong, keep attention or risk to send unuseful notification to other people

**Comment by bigbigmdm** [comment]
@Fantu, you wrote "the only possibility would be only if there are users who are or users who ask for it."

So if a normal user like @axet  sends a request to Debian about the need for this programme it should help?

In this case, how should this request be made and to which address should it be sent?

**Comment by Fantu** [comment]
Regarding individual users request there is only some more possibility that some DD will sponsor it if there is demand, but it could still be of little relevance in itself. But maybe also that any extra messages just to require adding would be more of a nuisance in some cases than a benefit.

**Comment by bigbigmdm** [comment]
@Fantu, you've completely confused me :)

**Comment by Fantu** [comment]
Sorry for my bad english and explain. I don't know if any user requests would be of benefit or on the contrary only a nuisance.

**Comment by bigbigmdm** [comment]
Thanks, @Fantu. I've already figured that out. But I don't know what to do next.
Make a new tag v1.1.11 and release `unstable`? Will that help in any way?

**Reply by Fantu** [reply]
New upstream version is not needed for new upload to mentors with target unstable, as wrote above following https://github.com/bigbigmdm/IMSProg/issues/19#issuecomment-1914447047

**Comment by bigbigmdm** [comment]
@Fantu, so it would be `imsprog (1.1.10-2) unstable; urgency=medium` with old tag `v1.1.10`?

**Comment by Fantu** [comment]
:( I already explained, should remain one entry of the initial release (short version, for details search in old messages, not here).
I already wrote what finalize d/changelog for release to unstable: `dch -r --distribution unstable`
And a temporary commit without push will be the best (to avoid multiple commits of "fake release"), for remove it if other changes are needed or push if it will be uploaded and accepted. But if you still have difficult with git usage push anyway, remember that if not upload next change must still be the "initial release" entry.
I am about to give up on continuing to write the same things over and over again :cry:

**Comment by bigbigmdm** [comment]
Forgive me, @Fantu! I just figured that the `unstable` release should complement the `UNRELEASED` release.

**Reply by Fantu** [reply]
Latest upload to mentors seems ok, but I don't see the RFS for it. You can use the autogenerated one from https://mentors.debian.net/sponsors/rfs-howto/imsprog/

**Comment by bigbigmdm** [comment]
Thank you, @Fantu ! I sent in this RFS. (Bug#1062599)
